src/autocomplete.js

```diff
@@ -502,6 +509,7 @@ yourlabs.Autocomplete.prototype.show = function(html) {
     // Show the inner and outer container only if necessary.
     if (!this.box.is(':visible')) {
         this.box.css('display', 'block');
+        this.fixPosition();
```

Comment:
Probably the display: block changes the position so we have to fix it again, but it would be nice to avoid calling it twice.